### PR TITLE
Issue 363 flags

### PR DESCRIPTION
**Pull request by kain88-de**
- correctly mark flag system tests. 
- fix one flag issue

*note*: these tests will be really nice to find other issue in the new topology system since I see failures with them unrelated to the pbc issue (i hope).

### CODE REVIEW

**Comment by richardjgowers** [comment]
So these tests won't work atm as the topology for this system doesn't have masses, but I think I'll add masses to the psf file/topology so these can stay as is. Then we won't have to skip them which is preferable.

**Comment by kain88-de** [comment]
Well most of them execute but the results check does vary slightly. it isn't just the masses.

File: testsuite/MDAnalysisTests/test_atomgroup.py [hunk 1]
```diff
@@ -2171,20 +2170,18 @@ def setUp(self):
...
         del self.ref_PBC
         del self.ag
 
-    # INVALID: no flags in new system
     def test_flag(self):
         # Test default setting of flag
         assert_equal(MDAnalysis.core.flags['use_pbc'], False)
 
-    # VALID
+    @skip
```
Comment (richardjgowers) [inline testsuite/MDAnalysisTests/test_atomgroup.py]: So we won't want the `skip` decorators if we intend to keep these tests though?

Comment (kain88-de) [inline testsuite/MDAnalysisTests/test_atomgroup.py]: Sorry they should be marked as invalid and then updated to run again.

Comment (kain88-de) [inline testsuite/MDAnalysisTests/test_atomgroup.py]: I marked then as invalid like the other tests

**Pull request merged**